projects/demo/src/modules/experimental/pin/examples/2/index.less

```diff
@@ -0,0 +1,24 @@
+:host {
+    display: flex;
+    gap: 2rem;
+    background: var(--tui-base-02);
+    box-shadow: 0 0 0 2rem var(--tui-base-02);
+    padding: 0.375rem 0 0 0.375rem;
+    margin-bottom: -0.375rem;
```

Comment:
The same question

Reply:
This is just a compensation for centered positioning of the pins. Alternatively we can absolutely position them and not have this weird compensation.